### PR TITLE
Event section 

### PR DESCRIPTION
**Pull request by leenaAhmed**

### CODE REVIEW

File: src/modules/events/components/eventCard/index.js [hunk 1]
```diff
@@ -1,7 +1,63 @@
...
 
-export default class EventCard extends Component {
-  render() {
-    return <div></div>;
-  }
+class EventCard extends Component{ 
+
+  render(){
+  
+    const {events} = this.props ;
```
Comment (dvmhmdsd) [inline src/modules/events/components/eventCard/index.js]: Events ==> Event, it's a single event.

File: src/modules/events/components/eventCard/index.js [hunk 2]
```diff
@@ -1,7 +1,63 @@
...
+
+    return (
+           <div className={styles["event-card_content"]}>
```
Comment (dvmhmdsd) [inline src/modules/events/components/eventCard/index.js]: `section` tag is more appropriate than `div`.

File: src/modules/events/components/eventCard/index.js [hunk 3]
```diff
@@ -1,7 +1,63 @@
...
+                        className={styles["activ-upcoming-event_icon"]}/></div>: ''}
+                  </p>
+                  <figcaption >
+                      <h3>
+                        <Link to={'/' + events.id} className={styles["event-card_title"]}>
+                          {events.title}
+                          </Link>
+                      </h3>
+                      <p className={styles["event-card_description"]}>
+                          {events.description}
```
Comment (dvmhmdsd) [inline src/modules/events/components/eventCard/index.js]: The description should be limited to a number of words, e.g: 20 words.

File: src/modules/events/components/eventCard/index.js [hunk 4]
```diff
@@ -1,7 +1,63 @@
...
+                      </p>
+                  </figcaption>
+              </figure>
+              <div class={styles["event-card_footer"]}>
```
Comment (dvmhmdsd) [inline src/modules/events/components/eventCard/index.js]: `footer` tag is more appropriate than `div`.

File: src/modules/events/components/eventsSection/style.module.css [hunk 5]
```diff
@@ -0,0 +1,25 @@
+.event-card_section{
+  margin-top: var(--section-y-spacing);
+  margin-bottom: var(--section-y-spacing);
```
Comment (dvmhmdsd) [inline src/modules/events/components/eventsSection/style.module.css]: `padding` is more appropriate like other sections.

**Review by dvmhmdsd — CHANGES_REQUESTED** [review]

File: src/modules/events/components/eventCard/index.js (hunk 3 above)
Comment (leenaAhmed) [reply src/modules/events/components/eventCard/index.js]: I'm used lines ellipsis to limited it

File: src/modules/events/components/eventsSection/index.js [hunk 6]
```diff
@@ -1,7 +1,45 @@
...
+   (<div className='center'> loading.... </div> ) 
+    return (
+      <section className={styles["event-card_section"]}>
+         <div className="container">
+            <h4 className="section_heading">our events</h4>
+            <div className="row">     
+               {EventsList}
+             </div>
+                <div className="row">
+                 <Link to='/events'
```
Comment (dvmhmdsd) [inline src/modules/events/components/eventsSection/index.js]: The colors of the text is not accessible on hover

File: src/modules/events/components/eventsSection/index.js [hunk 7]
```diff
@@ -1,7 +1,45 @@
...
+      return (
+        <div  key={eventsCard.id} className="col-lg-4 col-md-6 col-sm-6" >
+                 <EventCard events={eventsCard}  />
+        </div>)
+       }) :
+   (<div className='center'> loading.... </div> ) 
+    return (
+      <section className={styles["event-card_section"]}>
+         <div className="container">
+            <h4 className="section_heading">our events</h4>
```
Comment (dvmhmdsd) [inline src/modules/events/components/eventsSection/index.js]: `h2` is more appropriate than `h4` for semantics

**Review by dvmhmdsd — CHANGES_REQUESTED** [review]
- Resolve the errors in the console.

- Decrease the height of the image in the event card.

File: src/modules/events/components/eventCard/index.js (hunk 3 above)
Comment (dvmhmdsd) [reply src/modules/events/components/eventCard/index.js]: Great

**Comment by dvmhmdsd** [comment]
- The color of the hover of the title of the card is not accessible in dark mode, make it white better or just a border bottom.

- Also try to make the box-shadow from right and bottom only.

**Comment by dvmhmdsd** [comment]
Please solve the errors in the console

**Comment by dvmhmdsd** [comment]
Please solve the errors in the console !

File: src/modules/events/components/eventCard/index.js [hunk 8]
```diff
@@ -1,7 +1,69 @@
...
-  }
+class EventCard extends Component{ 
+
+  render(){
+  
+    const {event} = this.props ;
+     return (
+           <section className={styles["event-card_content"]}>
+              <figure className={styles["events-card_cover"]} key={event}>
+                  <Link to={'/' + event.id}>
```
Comment (esmail-elmoussel) [inline src/modules/events/components/eventCard/index.js]: to events/id

File: src/modules/events/components/eventCard/index.js [hunk 9]
```diff
@@ -1,7 +1,69 @@
...
+                        alt={`event ${event.title}`}
+                        src={event.cover}
+                        key={event.id} 
+                        />
+                  </Link>
+                  <p className={styles['activ-upcoming_event']}>
+                    {event.status === "upcoming"?<div>
+                     <FontAwesomeIcon icon={faClock}
+                        className={styles["activ-upcoming-event_icon"]}/></div>: ''}
+                  </p>
```
Comment (esmail-elmoussel) [inline src/modules/events/components/eventCard/index.js]: change the p tag to figure for example and the div tag to span to solve console error.

**Comment by dvmhmdsd** [comment]
Resolve conflicts please

**Pull request merged**